af_rest_appointment/models/af_appointment.py

```diff
@@ -251,14 +315,21 @@ def get_appointments(self, from_date, to_date, user = '', pnr = '', appointment_
     # /resource-planning/competencies/schedules
     def get_schedules(self, from_datetime, to_datetime, type_ids):
         """fetches schedules from Teleopti via IPF and creates calendar.schedule in odoo"""
-        client_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_id')
-        client_secret = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_secret')
-        af_environment = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_environment')
-        af_port = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_port')
-        af_url = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_url')
-        af_system_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_system_id')
-
-        if not (af_url or af_port or client_id or client_secret or af_environment or af_system_id):
+        client_id = self.env['ir.config_parameter'].sudo(
```

Comment:
This is almost unreadable. could you loook into any other way of getting these lines to go through Travis?